pset-functions.js

```diff
@@ -91,6 +127,22 @@ console.log('Problem 2:')
 
 // Add your code below this line
 
+const phraseString = "Humpty Dumpty sat on a wall Humpty Dumpty had a great fall"
+
+
+const phrase = phraseString.split(' ')
+
+
+function wordCount (phrase) {
+  const countArray = { }
```

Comment:
@kcycle the name chosen for this variable is a little confusing since the actual datatype you're creating is an object and not an array